### PR TITLE
Add otj-metrics-micrometer module

### PR DESCRIPTION
**Pull request by janardhan-chalo**

### CODE REVIEW

File: otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/CustomNameMapper.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    private static final List<String> tagsToAvoidInMetricPath = Collections.singletonList("exception");
+
+    private final String prefix;
+
+    public CustomNameMapper(String prefix) {
+        this.prefix = prefix;
+    }
+
+    @Override
+    public String toHierarchicalName(Meter.Id id, NamingConvention convention) {
```
Comment (mikebell90) [inline otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/CustomNameMapper.java:21]: Still not pluggable eh?

**Review by mikebell90 — APPROVED** [review]
You and @dkaukov need to add docs to README.

File: otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/WebMvcMetricsCustomNameConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.micrometer.core.instrument.config.MeterFilter;
+
+@Configuration
+public class WebMvcMetricsCustomNameConfiguration {
+
+    private static final String metricPrefix = "http.server.requests";
+    private static final String[] tagsToIgnore = {"exception", "outcome"};
```
Comment (dkaukov) [inline otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/WebMvcMetricsCustomNameConfiguration.java:12]: Why we want to ignore "exception"? We using it currently and it not causing cardinality problems,

Comment (janardhan-chalo) [reply otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/WebMvcMetricsCustomNameConfiguration.java:12]: ok I will remove it

Comment (janardhan-chalo) [reply otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/WebMvcMetricsCustomNameConfiguration.java]: Done

File: otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/CustomNameMapper.java (hunk 1 above)
Comment (janardhan-chalo) [reply otj-metrics-micrometer/src/main/java/com/opentable/metrics/micrometer/CustomNameMapper.java:16]: Meter filter makes this pluggable.

**Pull request merged**